azure-mgmt-graph-rbac/src/main/test/java/com/microsoft/azure/management/graphrbac/UsersTests.java

```diff
@@ -31,11 +31,14 @@ public void canCRUDUser() throws Exception {
         //LIST
         List<User> userList = graphRbacManager.users().list();
         Assert.assertNotNull(userList);
-        User user = graphRbacManager.users().define("jeffrolfnlu@hotmail.com")
+        User user = graphRbacManager.users().define("azuresdk@outlook.com")
```

Comment:
Can we delete this user after test is done? not sure if we can do so. But this test is not doing much once it run for first time as this user will always be there for ever.